pkg/agent/utils.go

```diff
@@ -375,6 +375,40 @@ func IsKubeletConfigFileEnabled(cs *datamodel.ContainerService, profile *datamod
 			IsKubernetesVersionGe(cs.Properties.OrchestratorProfile.OrchestratorVersion, "1.14.0"))
 }
 
+// IsKubeletClientTLSBootstrappingEnabled get if kubelet client TLS bootstrapping is enabled
+func IsKubeletClientTLSBootstrappingEnabled(cs *datamodel.ContainerService, profile *datamodel.AgentPoolProfile, kubeletClientTLSBootstrappingEnabled bool) bool {
+	if !kubeletClientTLSBootstrappingEnabled {
+		// toggle is off, we don't enable it
+		return false
+	}
+
+	if profile.TLSBootstrapToken == nil {
+		// agent node's TLS bootstrap token is not set
+		return false
+	}
+
+	if !datamodel.IsKubernetesVersionGe(cs.Properties.OrchestratorProfile.OrchestratorVersion, "1.4.0") {
```

Comment:
our aks supports k8s version > 1.15, so it's not necessary here to check version

Reply:
updated, PTAL